backend/inspirehep/search/factories/search.py

```diff
@@ -17,6 +17,12 @@
 
 
 def get_search_with_source(search):
+    request_fields = request.values.get("fields", "", type=str)
+    if request_fields:
+        request_fields = request_fields.split(",")
+        request_fields.append("control_number")
```

Comment:
is this a requirement? I would put something like `self` which is more API friendly.

Reply:
I have issues in invenio record fetcher when control_number is missing :/ so I just added it